packages/input-select/src/components/withImage.js

```diff
@@ -20,7 +20,7 @@ export default Input =>
         <LabelContainer className="labelContainer">
           {image && (
             <ItemImage
-              className="itemImage"
+              className="selectedItemImage"
```

Comment:
just changed the classname here... Is it enough?

Reply:
That's the pr purpose.

Reply:
Still didn't get based on the issue description. Can you elaborate more?

Reply:
Yes, for sure. In select inputs we have the dropdown items which are the options, and the item that was selected (if there is one). We have few different styles in both, mainly using the small size. The problem is that in both cases we use the same class, which is not used internally, was created to have access to the element style at implementation time. Applying styles in this class gave different behaviors between the selected item and the others. So, doesn't make any sense to use the same class on both, correct? That's why i created this simple PR.

Reply:
Ohhhh now i see, i thought it was about the selected image inside the dropdown items
Thanks!